prismic-model/js/popup-dialog.js

```diff
@@ -0,0 +1,19 @@
+// @flow
+
+import structuredText from './parts/structured-text';
+import text from './parts/text';
+import booleanNew from './parts/boolean-new';
```

Comment:
Prismic now has a built in `Boolean` type, where previously we were using a select with two options. Better name suggestions welcomed.

Reply:
I would call this one `boolean`, and rename the other one to something like `booleanDeprecated` to ensure we start using this one from now.